Changes

```diff
@@ -31,6 +31,9 @@ Working version
 
 ### Manual and documentation:
 
+- GPR#1741: manual, improve typesetting and legibility in HTML output
+  (steinuil)
```

Comment:
I would like to backport your change in the 4.07 branch, so that they effect the manual of the upcoming release. @steinuil, could you displace this Change entry to be part of the same section in the "OCaml 4.07" entry below?